packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1374,7 +1399,7 @@ abstract class BuildableElement extends Element {
 typedef Widget WidgetBuilder(BuildContext context);
 typedef Widget IndexedBuilder(BuildContext context, int index);
 
-// See _builder.
+// See ComponentElement._builder.
```

Comment:
True, but IDEs still decorate them even if they don't make it into the public docs, e.g. in Atom you should still be able to Ctrl + Click on the link.